grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingService.java

```diff
@@ -603,7 +602,7 @@ private HttpResponse serve0(ServiceRequestContext ctx, HttpRequest req,
                         ctx.setAttr(FramedGrpcService.RESOLVED_GRPC_METHOD, spec.method);
                         frameAndServe(unwrap(), ctx, grpcHeaders.build(),
                                       convertToJson(ctx, clientRequest, spec),
-                                      responseFuture, generateResponseBodyConverter(spec), jsonContentType);
+                                      responseFuture, generateResponseBodyConverter(spec), MediaType.JSON_UTF_8);
```

Comment:
nit: Let's leave a link to this https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter#sending-arbitrary-content that explains why we set JSON media type.